portfolio/pom.xml

```diff
@@ -1,53 +1,64 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Simply a question: Any reasons to switch the order of the two? To fix a problem, or just by accident?

Reply:
I think they might've switched when I ran a code cleanup in intellij